README.md

```diff
@@ -94,7 +94,7 @@ Please check the [API Reference](docs/sbom-tool-api-reference.md) document for a
 
 Once you have installed the command line tool for your OS, run the tool using this command:
 
-```
+```shell
```

Comment:
In this case, we want the < and > to be placeholders, not literals, so we want the generic formatting